src/components/KPop/KPop.vue

```diff
@@ -461,195 +336,208 @@ $kPopCaretOffset: 16px;
 
 /* Component styles */
 
-.k-popover, :deep(.k-popover) {
-  background-color: var(--kui-color-background, $kui-color-background);
-  border: var(--kui-border-width-10, $kui-border-width-10) solid var(--kui-color-border, $kui-color-border);
-  border-radius: var(--kui-border-radius-30, $kui-border-radius-30);
-  box-shadow: var(--kui-shadow, $kui-shadow);
+.k-popover {
```

Comment:
question: any idea why the deep selector was previously here?

Reply:
Not sure but I think we needed it for other components using KPop. I checked every component and they all look fine so let's roll with this for now.